persistence/src/main/java/edu/softserve/zoo/persistence/exception/PersistenceException.java

```diff
@@ -1,15 +1,14 @@
-package edu.softserve.zoo.exceptions.persistence;
+package edu.softserve.zoo.persistence.exception;
 
 import edu.softserve.zoo.exceptions.ApplicationException;
 import edu.softserve.zoo.exceptions.ExceptionReason;
 
 /**
- * <p>Base exception of the persistence module.
- * Reports about all kinds of problems.</p>
+ * <p>The base exception type for Persistence layer.</p>
  *
  * @author Bohdan Cherniakh
  */
-public class PersistenceException extends ApplicationException {
+public abstract class PersistenceException extends ApplicationException {
```

Comment:
this abstraction could be the convenient way to handle layer exceptions if we would need it